src/core/inlines.js

```diff
@@ -22,6 +22,82 @@ import { renderInlineCitation } from "./render-biblio";
 export const name = "core/inlines";
 export const rfc2119Usage = {};
 
+/**
+ * @param {string} matched
+ * @param {DocumentFragment} df
+ */
+function inlineRFC2119Matches(matched, df) {
+  matched = matched.split(/\s+/).join(" ");
+  df.appendChild(
+    hyperHTML`<em class="rfc2119" title="${matched}">${matched}</em>`
+  );
+  // remember which ones were used
+  rfc2119Usage[matched] = true;
+}
+
+/**
+ * @param {string} matched
+ * @param {DocumentFragment} df
+ */
+function inlineXrefMatches(matched, df) {
+  const ref = matched
+    .replace(/^\{{3}/, "")
+    .replace(/\}{3}$/, "")
```

Comment:
Maybe we can just `.slice(3, -3)` here because the assumption here is that the matched string is formed as `{{{ ... }}}`.

Reply:
ah, cute solution @saschanaz!

Reply:
done